### PR TITLE
Add media section

### PR DESCRIPTION
**Pull request by RazvanN7**

### CODE REVIEW

File: description.md [hunk 1]
```diff
@@ -13,6 +13,17 @@ Each section details how a specific type of content is to be developed, stored a
 
 ### Media
 
+The `media` content type refers to audio, visual or audiovisual educational materials.
+Images, videos, recordings, educational video games represent examples of such content.
```
Comment (teodutu) [inline description.md:17]: Nice to think about video games.

File: description.md [hunk 2]
```diff
@@ -13,6 +13,17 @@ Each section details how a specific type of content is to be developed, stored a
...
+
+The media content type represents an easier digestible format of presenting information.
```
Comment (teodutu) [inline description.md:19]:
```suggestion
The media content type represents an easily digestible format for presenting information.
```

File: description.md [hunk 3]
```diff
@@ -13,6 +13,17 @@ Each section details how a specific type of content is to be developed, stored a
...
+As such, it is typically used in conjunction with other content types, such as reading, slides or demos.
```
Comment (teodutu) [inline description.md:20]: Elaborate on this that media items are typically embedded within the text destined for reading, practice items, assignments or slides.

File: description.md [hunk 4]
```diff
@@ -13,6 +13,17 @@ Each section details how a specific type of content is to be developed, stored a
...
+
+When developing `media` items, it is important to follow the guidelines below:
+
+- use an open source or free development tool.
+- store both the output (the actual image, video etc.) and the editable building blocks.
```
Comment (teodutu) [inline description.md:25]: Not necessarily true. `.svg` images may serve as both the output and the editable building blocks (without requiring the `.drawio` files which are mostly the same thing).

Secondly, add a line about recordings and screenshots. There's nothing editable about them and unfortunately, they are to be stored as they are.

**Review by teodutu — CHANGES_REQUESTED** [review]

File: description.md (hunk 3 above)
Comment (RazvanN7) [reply description.md:20]: I did not elaborate on purpose because I was thinking this can be better explained at the "Teaching stages chapter". Of course we can elaborate on both sections

File: description.md [hunk 5]
```diff
@@ -13,6 +13,17 @@ Each section details how a specific type of content is to be developed, stored a
...
 
+The `media` content type refers to audio, visual or audiovisual educational materials.
+Images, videos, recordings, educational video games represent examples of such content.
+
+The media content type represents an easier digestible format of presenting information.
+As such, it is typically used in conjunction with other content types, such as reading, slides or demos.
+
+When developing `media` items, it is important to follow the guidelines below:
+
+- use an open source or free development tool.
```
[the remark this answers is not in the document]
Comment (RazvanN7) [reply description.md:24]: I tried to keep it as separated from actual technology as possible because this falls under the infrastructure/open source tools chapter.

File: description.md (hunk 4 above)
Comment (RazvanN7) [reply description.md:25]: >Not necessarily true. .svg images may serve as both the output and the editable building blocks (without requiring the .drawio files which are mostly the same thing).

That's just a particular case where the output and the editable building block are the same file.

Comment (RazvanN7) [reply description.md:25]: > Secondly, add a line about recordings and screenshots. There's nothing editable about them and unfortunately, they are to be stored as they are.

I will elaborate on this.

File: description.md (hunk 3 above)
Comment (teodutu) [reply description.md:20]: Does it have to do with teaching? It's about where the media items fit in, which is still part of the stored content.

File: description.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (Alex-deVis) [reply description.md:24]: I agree with @teodutu, providing some examples makes it easier for people to develop materials. Also, most of them are intuitive enough to be referrenced without a description.

File: description.md [hunk 6]
```diff
@@ -13,6 +13,17 @@ Each section details how a specific type of content is to be developed, stored a
...
+- store both the output (the actual image, video etc.) and the editable building blocks.
+
```
Comment (Alex-deVis) [inline description.md:26]: It is worth mentioning that slides are better than gifs when it comes to illustrating processes or sequential actions. Slides enable students to go at their own pace and also require no additional infrastructure.

File: description.md (hunk 1 above)
Comment (razvand) [reply description.md:17]: Recordings are still videos, right? I would word it as "Images, video and audio files, educational video games ..."

File: description.md [hunk 7]
```diff
@@ -13,6 +13,46 @@ Each section details how a specific type of content is to be developed, stored a
 
 ### Media
 
+The `media` content type refers to audio, visual or audiovisual educational materials.
+Images, videos, recordings, educational video games represent examples of such content.
+
+The media content type represents an easily digestible format for presenting information.
+As such, media is typically embbeded within the text that is destined reading, slides, demos, practice items and projects.
+
+When developing `media` items, it is important to follow the guidelines below:
```
Comment (razvand) [inline description.md:22]: Why is media marked with backticks here but not elsewhere? I would remove backticks.

File: description.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (razvand) [reply description.md:24]: Agree with @teodutu and @Alex-deVis. Make it concrete, give the reader assistance. Examples are always a good idea.

File: description.md (hunk 4 above)
Comment (razvand) [reply description.md:25]: I would word it as "being contributable / editable" - where possible (screenshots can't fit this). Such that someone could then get the media content and be able to improve it according to the 5R content properties. Only then, I would mention storing both the original content and the output (where possible).

File: description.md [hunk 8]
```diff
@@ -13,6 +13,46 @@ Each section details how a specific type of content is to be developed, stored a
...
+
+- use an open source or free development tool.
+- store both the output (the actual image, video etc.) and the editable building blocks.
+
+#### Images
+
+For the creation of images and diagrams we use [draw.io](draw.io).
```
Comment (razvand) [inline description.md:29]:
```suggestion
For the creation of images and diagrams we use [draw.io](https://draw.io).
```

File: description.md [hunk 9]
```diff
@@ -13,6 +13,46 @@ Each section details how a specific type of content is to be developed, stored a
...
+`draw.io` is a free, easy to use tool which is available in the browser.
+The resulting images can be exported in different formats, such as PDF, PNG or SVG.
+When storing images, it is important that both the source file (with the extension `drawio` or `svg`) and the immutable output (PDF, PNG, JPG) are stored.
```
Comment (razvand) [inline description.md:32]: Mention why this is important. In general, argue our decisions.

File: description.md [hunk 10]
```diff
@@ -13,6 +13,46 @@ Each section details how a specific type of content is to be developed, stored a
...
+
+When developing `media` items, it is important to follow the guidelines below:
+
+- use an open source or free development tool.
+- store both the output (the actual image, video etc.) and the editable building blocks.
+
+#### Images
+
+For the creation of images and diagrams we use [draw.io](draw.io).
+`draw.io` is a free, easy to use tool which is available in the browser.
```
Comment (razvand) [inline description.md:30]: Argue why having it as a browser helps, that it exports SVG and that it contains many useful icons / basic blocks. Hence, our decision to use it.

File: description.md [hunk 11]
```diff
@@ -13,6 +13,46 @@ Each section details how a specific type of content is to be developed, stored a
...
+The resulting images can be exported in different formats, such as PDF, PNG or SVG.
+When storing images, it is important that both the source file (with the extension `drawio` or `svg`) and the immutable output (PDF, PNG, JPG) are stored.
+Note that other tools for the generation of images may be used as long as they are free and provide the source file.
+
```
Comment (razvand) [inline description.md:34]: Mention about screenshots.

File: description.md [hunk 12]
```diff
@@ -13,6 +13,46 @@ Each section details how a specific type of content is to be developed, stored a
...
+#### Audio-video recordings
```
Comment (razvand) [inline description.md:35]:
```suggestion
#### Audio-video Recordings
```

File: description.md [hunk 13]
```diff
@@ -13,6 +13,46 @@ Each section details how a specific type of content is to be developed, stored a
...
+In terms of the tools that can be used to create such recordings, given the variety of options, we do not give any specific recommendations.
+Any device that can record at a decent quality is fine.
+
+There is a particular type of video that can be created without using a camera.
+By putting multiple similar images in a slideshow and automating the presentation, the video effect can be achieved.
+This sort of video is useful for educational material that presents processes or phenomenons step by step.
+To create such videos, we need to first create the images and then bundle them up using [ffmpeg](https://ffmpeg.org/).
+In this situation the image source files, the actual images and the video need to be stored in the content repository.
+
+#### Interactive applications
```
Comment (razvand) [inline description.md:49]:
```suggestion
#### Interactive Applications
```

File: description.md [hunk 14]
```diff
@@ -13,6 +13,46 @@ Each section details how a specific type of content is to be developed, stored a
...
+
+For the creation of images and diagrams we use [draw.io](draw.io).
+`draw.io` is a free, easy to use tool which is available in the browser.
+The resulting images can be exported in different formats, such as PDF, PNG or SVG.
+When storing images, it is important that both the source file (with the extension `drawio` or `svg`) and the immutable output (PDF, PNG, JPG) are stored.
+Note that other tools for the generation of images may be used as long as they are free and provide the source file.
+
+#### Audio-video recordings
+
+Audio or video content is by its nature hard to modify.
```
Comment (razvand) [inline description.md:37]: I would do a classification of audio-video content and correlate it with use cases. Otherwise, it simply appears as a listing of potential approaches, but the reader wouldn't know what to use where.

I would add:
- Animations: succession of images that show a process, a set of steps - these can be created with an image software and then an image-tying software such as ffmpeg
- Screencasts: someone records their screen, it requires a screen recording software - give examples, this typically screencasts applications (terminal, browser, simulators)
- Actual recordings: a live recording with a camera of someone giving a talk or detailing a concept; this can also be, in certain cases, an audio recording only

I would then add sections on each of those and detail the actual methodology for producing these types of contents, together with sample tools they would use for them.

File: description.md [hunk 15]
```diff
@@ -13,6 +13,46 @@ Each section details how a specific type of content is to be developed, stored a
...
+Typically, if a video/audio recording requires alterations, it is re-recorded.
+As such, the video/audio recording is stored as is and if modifications are required, it needs to be recreated.
+In terms of the tools that can be used to create such recordings, given the variety of options, we do not give any specific recommendations.
+Any device that can record at a decent quality is fine.
+
+There is a particular type of video that can be created without using a camera.
+By putting multiple similar images in a slideshow and automating the presentation, the video effect can be achieved.
+This sort of video is useful for educational material that presents processes or phenomenons step by step.
+To create such videos, we need to first create the images and then bundle them up using [ffmpeg](https://ffmpeg.org/).
+In this situation the image source files, the actual images and the video need to be stored in the content repository.
```
Comment (razvand) [inline description.md:47]: I don't think we are storing the video in the repository, do we? @teodutu ?

File: description.md (hunk 14 above)
Comment (razvand) [reply description.md:37]: Either here, when presenting the media content types, or in each section, provide use cases for each of those. The reader should have answers to questions:
- What should I do if I want to do X (with respect to media content)? (get from use case to tool)
- What should I use this media content type for? (get from tool to use case)

File: description.md (hunk 13 above)
Comment (razvand) [reply description.md:49]: Not sure if this should be part of the media section. This is more of "support software". But, then again, I can't think of a better place to add this content type.

File: description.md [hunk 16]
```diff
@@ -13,6 +13,46 @@ Each section details how a specific type of content is to be developed, stored a
...
+
+#### Interactive applications
+
+Some educational scenarios involve the learner to interact with a specific application.
+The application can be a video game, a web page or even a simple computer program that simulates a given context.
+For such situations, the code of the application should be stored alongside the information regarding the tools that are necessary to build the application.
```
Comment (razvand) [inline description.md:53]: This I don't get. You mean the code that's used by the application to run the demo / scenario, right? Not the application code. I would reword it to make it more clear.

**Review by razvand — CHANGES_REQUESTED** [review]
